src/language/de/products/public.php

```diff
@@ -22,17 +23,27 @@
 'TYPE_HARDWARE' => 'Hardware',
 'TYPE_MUSIC' => 'Musik',
 'TYPE_MOVIE' => 'Film',
-'TYPE_BOOK' => 'Literatur'
+'TYPE_BOOK' => 'Literatur',
+
+/* Neue Variabeln mit Präfixe */
+'PRODUCTS_TYPES_ALL' => 'Alle',
+'PRODUCTS_TYPES_GENERAL' => 'Allgemein',
+'PRODUCTS_TYPES_LITERATURE' => 'Literatur',
+'PRODUCTS_TYPES_MUSIC' => 'Musik',
+'PRODUCTS_TYPES_MOVIE' => 'Film',
+'PRODUCTS_TYPES_VIDEOGAME' => 'Videospiel',
+'PRODUCTS_TYPES_SOFTWARE' => 'Software',
+'PRODUCTS_TYPES_HARDWARE' => 'Hardware',
 );
 
 
 $lang['fields'] = array(
 'PRODUCT' => 'Produkt',
-'RELEASE' => 'Ver�ffentlichung',
+'RELEASE' => 'Veröffentlichung',
```

Comment:
Ich vermute der Zeilenumbruch war nicht konsistent und wurde aktualisiert, deshalb wird das als veränderte Zeile markiert, denn am Inhalt hat sich nichts geändert. Wahrscheinlich sollten wir erst einmal den gesamten Quellcode vereinheitlichen und solche Fehler beseitigen, ansonsten wird das immer wieder in den Diffs auftauchen und Änderungen anzeigen die gar keine sind.